src/transaction.rs

```diff
@@ -75,6 +75,13 @@ pub enum Instruction {
     NewVote(Vote),
 }
 
+pub type Height = u64;
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Default)]
+pub struct LastId {
```

Comment:
Instead of adding this to the transaction, how about the server maintains a lookup table from recent PoH hashes to its entry_height?

Reply:
that would only help with the transaction encoding.  When you query your account, you would need to query the server for the Hash + Height, so you can create a transaction at a version that is greater then whats in the account.